metpy/calc/basic.py

```diff
@@ -16,7 +16,7 @@
 
 import numpy as np
 
-from ..constants import g, omega, Rd
+from ..constants import g, omega, Rd, Re, G, me
```

Comment:
The checker doesn't like the order on these; it's alphabetization says:
"G, g, me, omega, Rd, Re"